pkg/products/rhsso/userHelper.go

```diff
@@ -0,0 +1,43 @@
+package rhsso
+
+import (
+	"context"
+	"fmt"
+	keycloak "github.com/keycloak/keycloak-operator/pkg/apis/keycloak/v1alpha1"
+	usersv1 "github.com/openshift/api/user/v1"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// Helper for User associated functions
+
+const (
+	updateProfileAction = "UPDATE_PROFILE"
+)
+
+func GetUserEmailFromIdentity(ctx context.Context, serverClient k8sclient.Client, user usersv1.User, ns string) (string, error) {
+	email := ""
+
+	// User can have multiple identities
+	for _, identityName := range user.Identities {
+		identity := &usersv1.Identity{}
+		err := serverClient.Get(ctx, k8sclient.ObjectKey{Name: identityName, Namespace: ns}, identity)
```

Comment:
Identities are not namespaced actually, so we don't need the `ns` var/param in this function. 
https://github.com/openshift/api/blob/d0b31d707c464221d1eb24846b0d0bbe57040102/user/v1/types.go#L47

Reply:
Thanks, good catch ! I've removed requiring passing / using the namespace name to the associated functions